documentation/modules/post/networking/gather/enum_brocade.md

```diff
@@ -70,7 +70,7 @@ verbose => true
 [+] ENCRYPTED SNMP community $MlVzZCFAbg== with permissions ro
 [+] ENCRYPTED SNMP community $U2kyXj1k with permissions rw
 [*] Post module execution completed
-msf5 post(brocade/gather/enum_brocade) > loot
+msf5 post(networking/gather/enum_brocade) > loot
```

Comment:
Woops upon further review this documentation doesn't show the `run` command at all here. Could we fix this up?

Reply:
oops!